index.bs

```diff
@@ -638,16 +638,17 @@ dictionary FocusableAreasOptions {
     FocusableAreaSearchMode mode;
 };
 
-dictionary SelectSpatNavBestCandidateOptions {
+dictionary SpatNavSearchOptions {
     required SpatialNavigationDirection dir;
-    required sequence&lt;Node> candidates;
+    sequence&lt;Node>? candidates;
+    Node? container;
 };
 
 partial interface Element {
     Node getSpatnavContainer();
     sequence&lt;Node> focusableAreas(optional FocusableAreasOptions arg);
-    Node? selectSpatNavBestCandidate(SelectSpatNavBestCandidateOptions arg);
-    Node? sequentialNavSearch(SequentialNavigationDirection dir);
+    Node? spatNavSearch(SpatNavSearchOptions arg);
```

Comment:
As said in #37, I really think these two methods should be merged to one method.

In Blink we've got the WebFocusType-enum that contains both sequential and spatial navigation "directions". Why not here?

Reply:
Sure!